### DISCUSSION TITLE
Enter minimum and maximum price in stable version

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#7947] [Q&A] [answered]
Hi,
   I'm using stable version and just realize that there is no way to enter/override minum and maximum price, how should this be done now? Thanks

**Comment by SchrodingersGat** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/975cda75-600e-464c-9ddf-09f5ad01ffbe)
